### DISCUSSION TITLE
【2023.7.9】OpenKF community meeting notes

### DISCUSSION THREAD

**Discussion by cubxxw** [openimsdk/openkf#78] [meeting]
**Conference link：**
> **Note**: Every two weeks at 2023/07/09 20:00-21:00 pm
https://meeting.tencent.com/dm/4B491joNSBnQ 📱#Tencent Conference：681-525-631

A discussion session about v0.1.0

**If you have a topic to discuss, please comment below.**

**Comment by IRONICBo** [comment]
1. Hook path matching.
2. Parse dao template and support more type.
3. Trace context route.
4. Interact with OpenIMServer.
5. Docker containers for each componests.
6. System extension APIs.
